pyface/qt/__init__.py

```diff
@@ -21,63 +21,21 @@
 ]
 
 
-def prepare_pyqt4():
-    """ Set PySide compatible APIs. """
-    # This is not needed for Python 3 and can be removed when we no longer
-    # support Python 2.
-    try:
-        # required for PyQt >= 4.12.2
-        from PyQt4 import sip
-    except ImportError:
-        import sip
-    try:
-        sip.setapi("QDate", 2)
-        sip.setapi("QDateTime", 2)
-        sip.setapi("QString", 2)
-        sip.setapi("QTextStream", 2)
-        sip.setapi("QTime", 2)
-        sip.setapi("QUrl", 2)
-        sip.setapi("QVariant", 2)
```

Comment:
This code path still gets run in Python 3 with PyQt. The outcome of this import side effect may not have been well tested, and it would be difficult to track down to this change if this removal should cause issues.
Are there other reasons for removing this apart from the comment and message surrounding it being Python 2 specific?

Reply:
No, it was simply removed because of the comment and Rahul saying we might be able to remove it on the original issue. To prevent potential additional headaches from its removal, I will add it back in - thanks!

Reply:
I see. Yes please add it back in. Thank you!
Finger-crossed we won't need to support PyQt4 for much longer so this function is on already the death row - just not quite there yet :)

Reply:
Should I add back in the places where it was called?  The one I'm most unsure about is the one below that was on line 61 as that if statement only occurs if python2 is being used explicitly.

Reply:
You're right - I think the one under `if qt_api == "pyqt" and sys.version_info[0] <= 2:` can be safely removed.